packages/osc-ui/src/components/Button/button.scss

```
@@ -547,18 +603,32 @@
         flex-wrap: wrap;
         align-items: center;
 
+        &--row {
+            flex-direction: row;
+        }
+
         &--column {
             flex-direction: column;
             align-items: flex-start;
         }
 
+        &--none {
+            gap: none;
```

`none` doesn't exist as a property for `gap` should be `0` instead
```suggestion
            gap: 0;
```